plugins/org.jkiss.dbeaver.ui.editors.image/src/org/jkiss/dbeaver/ui/controls/imageview/NativeImageEditor.java

```diff
@@ -31,15 +31,14 @@
 /**
  * Image editor, based on image viewer.
  */
-public class ImageEditor extends ImageViewer {
+public class NativeImageEditor extends NativeImageViewer {
```

Comment:
`Native` is a bad tern in this context. Maybe `SWT` or `Embedded`, otherwise the term is too broad